_posts/2025-05-16-perforce-eula.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: The Perforce EULA and impact on Vox Pupuli
+date: 2025-05-16
+github_username: bastelfreak
+---
+
+In November 2024, [Perforce announced](https://www.puppet.com/blog/open-source-puppet-updates-2025) that they will stop providing most tools & projects as open source.
+
+> In early 2025, Puppet will begin to ship any new binaries and packages developed by our team to a private, hardened, and controlled location.
+> Community contributors will have free access to this private repo under the terms of an End-User License Agreement (EULA) for development use. There will be no license changes for the open source version of Puppet.
+
+As the top community contributors to the Puppet module ecosystem and the maintainers of much of the Puppet module testing pipeline tooling, Vox Pupuli has significant expertise in developer experience.
+The [Vox Pupuli PMC](https://voxpupuli.org/elections/) immediately reached out to Perforce to discuss the [provided](https://github.com/voxpupuli/community-triage/issues/29) "[PUPPET® CORE DEVELOPER PROGRAM LICENSE AGREEMENT](https://www.perforce.com/system/files/2025-02/Puppet-Core-Developer-Program-License-Agreement.pdf)" in hopes of improving it and reducing the friction to contributing Puppet modules.
+
+As of today (2025-05-16), Vox Pupuli is unable to sign the current version of the Puppet Core Developer EULA.
```

Comment:
Maybe `the Vox Pupuli community has reached the conculsion that we are unable to sign`.  It sounds a bit like they did something today that caused us to not sign it.